plugins/catalog-backend/src/validation/KubernetesValidatorFunctions.test.ts

```diff
@@ -83,6 +83,25 @@ describe('KubernetesValidatorFunctions', () => {
     expect(KubernetesValidatorFunctions.isValidObjectName(value)).toBe(matches);
   });
 
+  it.each([
+    [7, false],
+    [null, false],
+    ['', false],
+    ['a', true],
+    ['AZ09', false],
+    ['a'.repeat(63), true],
+    ['a'.repeat(64), false],
+    ['a/b', false],
+    ['a-b', true],
+    ['-a-b', false],
+    ['a-b-', false],
+    ['a--b', false],
+    ['a_b', false],
+    ['a.b', false],
+  ])('isValidNamespace', (value, matches) => {
```

Comment:
```suggestion
  ])('isValidNamespace %p ? %p', (value, matches) => {
```
makes args to be presented in test name
https://jestjs.io/docs/en/api#1-testeachtablename-fn-timeout